generator/README.md

```diff
@@ -10,7 +10,7 @@ Due to the dynamic dependency on NetSNMP, you must build the generator yourself.
 ```
 sudo apt-get install build-essential libsnmp-dev # Debian-based distros
 # OR
```

Comment:
Can you remove this line?

Reply:
The line with the OR

Reply:
What if we did
# Debian-based distros
apt-get command
# RHEL-based distros
yum command

That way, it doesn't read like both commands should be executed

Reply:
That sounds reasonable to me.